src/vehicle.cpp

```diff
@@ -41,6 +41,13 @@
 #include <numeric>
 #include <algorithm>
 
+/**
+ *  Constant use to convert between wheel and engine rpm
+ *  1:3.7 is a fairly typical differential gearing
+ *  0.15 is a magic constant based upon ~15" tyres
+ */
+static constexpr float wheel_diff = 3.7 * 0.15;
```

Comment:
That could work but for now all wheels should be the same whilst we get the rest of the implementation worked out. Huge wheels with good traction but poor top speed could be a good later addition.